docs/troubleshooting.md

```diff
@@ -130,3 +130,183 @@ agent:
+    POD_NAME=splunk-otel-collector-agent-6llkr
+    # Verify the pod has proper stats with this command and sample output.
+    kubectl get --raw "/api/v1/nodes/"${NODE_NAME}"/proxy/stats/summary" | jq '.pods[] | select(.podRef.name=='\"$POD_NAME\"') | {"pod": {"name": .podRef.name, "cpu": .cpu, "memory": .memory, "network": .network}} | del(.pod.network.interfaces)'
+    {
+      "pod": {
+        "name": "splunk-otel-collector-agent-6llkr",
+        "cpu": {
+          "time": "2022-05-20T18:38:47Z",
+          "usageNanoCores": 10774467,
+          "usageCoreNanoSeconds": 1709095026234
+        },
+        "memory": {
+          "time": "2022-05-20T18:38:47Z",
+          "availableBytes": 781959168, # Could be absent if pod memory limits were missing.
+          "usageBytes": 267563008,
+          "workingSetBytes": 266616832,
+          "rssBytes": 257036288,
+          "pageFaults": 0,
+          "majorPageFaults": 0
+        },
+        "network": {
+          "time": "2022-05-20T18:38:55Z",
+          "name": "eth0",
+          "rxBytes": 105523812442,
+          "rxErrors": 0,
+          "txBytes": 98159696431,
+          "txErrors": 0
+        }
+      }
+    }
+    ```
+    </details>
+
+    <details>
+    <summary>3) Verify a container's stats</summary>
+
+    ```
+    # Get the names of the containers in your pod.
+    kubectl get --raw "/api/v1/nodes/"${NODE_NAME}"/proxy/stats/summary" | jq '.pods[] | select(.podRef.name=='\"$POD_NAME\"') | .containers[].name'
+    # Pick a container to evaluate and set it's name to an enviroment variable.
+    CONTAINER_NAME=otel-collector
+    # Verify the container has proper stats with this command and sample output.
+    kubectl get --raw "/api/v1/nodes/"${NODE_NAME}"/proxy/stats/summary" | jq '.pods[] | select(.podRef.name=='\"$POD_NAME\"') | .containers[] | select(.name=='\"$CONTAINER_NAME\"') | {"container": {"name": .name, "cpu": .cpu, "memory": .memory}}'
+    {
+      "container": {
+        "name": "otel-collector",
+        "cpu": {
+          "time": "2022-05-20T18:42:15Z",
+          "usageNanoCores": 6781417,
+          "usageCoreNanoSeconds": 1087899649154
+        },
+        "memory": {
+          "time": "2022-05-20T18:42:15Z",
+          "availableBytes": 389480448, # Could be absent if container memory limits were missing.
+          "usageBytes": 135753728,
+          "workingSetBytes": 134807552,
+          "rssBytes": 132923392,
+          "pageFaults": 93390,
+          "majorPageFaults": 0
+        }
+      }
+    }
+    ```
+    </details>
+
+### Reported incompatible Kubernetes and container runtime issues
+
+- Note: Managed Kubernetes services might use a modified container runtime,
+  the service provider may have applied custom patches or bug fixes that aren't
+  present within an unmodified container runtime.
+- Kubernetes 1.21.0-1.21.11 using containerd
+  - Issues:
+    - Memory and network stats can be missing.
```

Comment:
Can you provide any example metrics and their mapped or translated equivalents or are there too many (unknowns)?

Reply:
Added more details about what metrics are affected. Added stat name to metric name mapping examples.